syncer/syncer_test.go

```diff
@@ -589,6 +589,7 @@ func TestSync_Cancel(t *testing.T) {
 	}()
 
 	err := syncer.Sync(ctx, -1, 1200)
+	fmt.Println(err)
```

Comment:
nit: is this needed?

Reply:
good catch, I was using that line to debug 😅